config.sh

```diff
@@ -49,17 +49,16 @@ function pre_build {
         BUILD_PREFIX=`dirname $(dirname $(which python))`
         PKG_CONFIG_PATH="$BUILD_PREFIX/lib/pkgconfig"
     fi
-    if [[ $MACOSX_DEPLOYMENT_TARGET != "11.0" ]]; then
-		build_simple xcb-proto 1.14.1 https://xcb.freedesktop.org/dist
-		if [ -n "$IS_MACOS" ]; then
-			build_simple xproto 7.0.31 https://www.x.org/pub/individual/proto
-			build_simple libXau 1.0.9 https://www.x.org/pub/individual/lib
-			build_simple libpthread-stubs 0.4 https://xcb.freedesktop.org/dist
-		else
-			sed -i s/\${pc_sysrootdir\}// /usr/local/lib/pkgconfig/xcb-proto.pc
-		fi
-		build_simple libxcb $LIBXCB_VERSION https://xcb.freedesktop.org/dist
-    fi
+	build_simple xcb-proto 1.14.1 https://xcb.freedesktop.org/dist
```

Comment:
There's a mix of space and tab indents, shall we switch to spaces and maybe add a `.editorconfig`?

https://github.com/python-pillow/Pillow/blob/master/.editorconfig

Reply:
Ok, I replaced the tabs with spaces, and created #201